config/database.yml.dist

```diff
@@ -1,27 +0,0 @@
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
-  adapter: sqlite3
-  database: db/test.sqlite3
-  pool: 5
-  timeout: 5000
-
-development:
-  adapter: postgresql
-  database: bfr_webapp_db
-  username: bfr_webapp_user
-  password: CHANGEME
-  host: localhost
-  pool: 5
-  timeout: 5000
-
-production:
-  adapter: postgresql
-  database: bfr_webapp_db
-  username: bfr_webapp_user
-  password: CHANGEME
-  host: localhost
-  pool: 5
-  timeout: 5000
-
```

Comment:
Commit message says:

> Change test database from sqlite3 to psql

I would have expected this commit to change the `test` section of this file, moving from `sqlite3` to `postgresql`, but leave the rest of the file alone.

I think it would be good to keep this file around for now, since it's used as kind of a template for people getting the project running locally. The readme has a step: "[Next, copy /config/database.yml.dist to /config/database.yml and make any necessary changes](https://github.com/boulder-food-rescue/food-rescue-robot#database)".